### ISSUE TITLE
How about evaluation in Kinetics?

### ISSUE THREAD

**Issue by rimchang** [qijiezhao/pseudo-3d-pytorch#3]
How about evaluation in Kinetics?
and Is there some image normalization?
Is normalization same Resnet in pytorch?

**Comment by rimchang** [comment]
sorry, It is just converted model from original repo.

**Comment by qijiezhao** [comment]
1) I recently did some other work, this process(evaluate on Kinetics) will implement not long later.
2) Image normalization is optional.  Because you can keep training in this setting. 
3) yeah, both could by the method: torchvision.transforms.Normalize()
